### ISSUE TITLE
Two Options on the Git 3 Logo to pick from

### ISSUE THREAD

**Issue by Imperiumbrands** [Imperiumbrands/Git3LogoDesign#1]
![Logo-05](https://user-images.githubusercontent.com/66221152/144683382-c1d074bd-02e2-4243-82f0-171441ed22b3.png)
![Logo-06](https://user-images.githubusercontent.com/66221152/144683387-07c6294b-1f4c-475d-be0e-b20729c80ecf.png)
![Logo-07](https://user-images.githubusercontent.com/66221152/144683388-69ecabc3-c1d1-47b1-aa5b-487fa2c3fda3.png)
![Logo-02](https://user-images.githubusercontent.com/66221152/144683391-be4f1648-40db-47bb-a0ae-64ad80ad365d.png)
![Logo-03](https://user-images.githubusercontent.com/66221152/144683392-f64500f7-af90-4024-b488-290cad0280b2.png)
![Logo-04](https://user-images.githubusercontent.com/66221152/144683395-f643f1aa-0ffa-4146-8068-552fbac8723f.png)

**Comment by Imperiumbrands** [comment]
When you pick from the options, I will send you the pack via email.

**Comment by jacekv** [comment]
I like number 3 and 6. Could we try something regarding the Color? Like a black background and gold for the logo?

**Comment by Imperiumbrands** [comment]
> I like number 3 and 6. Could we try something regarding the Color? Like a black background and gold for the logo?

Oh yes, sure.

**Comment by Imperiumbrands** [comment]
![Git #-09](https://user-images.githubusercontent.com/66221152/144737086-984a69d8-5cde-4fee-bb8c-ab6ba741ecab.png)
![Git #-10](https://user-images.githubusercontent.com/66221152/144737089-18742b49-ef0b-482c-93a2-e4f527fe2f04.png)
![Git #-11](https://user-images.githubusercontent.com/66221152/144737091-23f40065-d039-4e05-96ad-b67addd532c5.png)

**Comment by Imperiumbrands** [comment]
![Git 3-09](https://user-images.githubusercontent.com/66221152/144737119-1f0785f0-58ad-47c2-8b9d-74298c458557.png)
![Git 3-10](https://user-images.githubusercontent.com/66221152/144737145-89669c8c-39f9-4027-8157-3a10c960bcbd.png)
![Git 3-11](https://user-images.githubusercontent.com/66221152/144737150-3f453e0c-7b29-472d-af35-55737dc4a286.png)

**Comment by jacekv** [comment]
I like that :) I will accept this as a submission :)

**Comment by Imperiumbrands** [comment]
> I like that :) I will accept this as a submission :)

Great!
Hoping to hear from you soon.

**Comment by jacekv** [comment]
Will you send it via email or how shall we do it?

**Comment by Imperiumbrands** [comment]
> Will you send it via email or how shall we do it?

Yeah. Email is the best option so as to get to send the PNG, pdf and SVGs.


I'm still trying to understand both GitHub and Gitcoin.
Complex interface (might have to propose a GitHub section for designers haha).

**Comment by jacekv** [comment]
I know how you feel :D
What is the problem? Maybe I can help.

**Comment by Imperiumbrands** [comment]
> I know how you feel :D What is the problem? Maybe I can help.

Everything haha.
For a non-programmer it's a bit of chaos.
Where to upload what?
Where to write what?
Where to look for what?

I wasn't even sure I did this (the whole post) right until I saw your comment 🤣

**Comment by Imperiumbrands** [comment]
By the way, I just shit you a connection request on LinkedIn https://www.linkedin.com/in/adetayo-adebajo-367aa7162

**Comment by jacekv** [comment]
Nice, accepted your request :) 
You have submitted something, so that so far. Once you send me the files I will be able to make the payout to your submission and you should be fine :)

**Comment by Imperiumbrands** [comment]
> Nice, accepted your request :) You have submitted something, so that so far. Once you send me the files I will be able to make the payout to your submission and you should be fine :)

Yeah 😂
I'll get the files to you soon via email.

**Comment by ghost** [comment]
![Logo](https://user-images.githubusercontent.com/62105597/150680738-66b6def4-9172-40c5-844a-70086977e008.png)
